great-idea-website/css/index.css

```diff
@@ -63,4 +63,156 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	width: 900px;
+	margin: auto;
+	border: 1px solid black;
```

Comment:
This is a *style opinion*, as in you can disregard it if you like, your code is awesome either way.

If you notice, a lot of the content in the container don't have any padding between it and the container. This wasn't noticeable before because there wasn't any border, so the whitespace around the page kind of acted like padding. But now with the border, it looks the content is pinned straight up to the wall so to speak, so adding a little padding might make the page more visually pleasing.